cmd/crates/soroban-spec-typescript/fixtures/ts/src/invoke.ts

```diff
@@ -54,7 +53,10 @@ export class NotImplementedError extends Error { }
  * @returns The transaction response, or the simulation result if signing isn't required.
  */
 export async function invoke({ method, args = [], fee = 100, signAndSend = false }: InvokeArgs): Promise<(TxResponse & { xdr: string }) | Simulation> {
-  const account = await getAccount()
+  const freighterAccount = await getAccount()
+
+  // use a placeholder account if not yet connected to Freighter so that view calls can still work
+  const account = freighterAccount ?? new SorobanClient.Account('GBZXP4PWQLOTBL3P6OE6DQ7QXNYDAZMWQG27V7ATM7P3TKSRDLQS4V7Q', '0')
```

Comment:
Non urgent followup: Not a big deal since the seq num is zero this tx will be unusable and I assume no one has the priv key, but to communicate even clearer that this account is meaningless I'd replace it with the zero account.

Reply:
This was just meant for simulation only calls. Does the sequence number matter?

Reply:
Yes, I'd love to replace it with the zero account. I haven't seen that around yet and don't see it documented at soroban.stellar.org/docs. Is this correct?

```
G0000000000000000000000000000000000000000000000000000000
```

Reply:
```rust
use stellar_strkey::ed25519::PublicKey;
fn main() {
    let zero = PublicKey([0; 32]);
    println!("{zero}");
}
```
```
GAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAWHF
```

Reply:
strkeys are base32 encoded. Lots of zeros encode into AAAAAAAAAA...

![](https://media2.giphy.com/media/Sm8Ec3ddRWXa8/giphy.gif?cid=ecf05e47pnrmeuj3cim98kvmgi003wnuud0felcqpln2154s&ep=v1_gifs_related&rid=giphy.gif&ct=g)